requirements-test.txt

```diff
@@ -7,5 +7,5 @@ pytest-xdist==1.31.0
 pytest==5.3.5
 redis==3.4.1
 requests==2.23.0
-sentry_sdk==0.14.3
+git+git://github.com/getsentry/sentry-python@e234998ae82a9cffa6fb3718801c55ba24a86bab#egg=sentry_sdk
```

Comment:
@jan-auer I'm pinning this to https://github.com/getsentry/sentry-python/commit/e234998ae82a9cffa6fb3718801c55ba24a86bab . I've confirmed with @untitaker that we can just pin this for the test python dependencies.

Reply:
yeah just for context I want to scale down on the py sdk releases I do because customers may be get pinged via dependabot for those and it may be kinda annoying. I already pushed out two releases in the past 5 days

Reply:
Looks like the only APIs we need are for tests and can be achieved with helper functions in our tests, too?